checks.d/mysql.py

```diff
@@ -385,9 +385,8 @@ def _get_server_pid(self, db):
         if pid is None:
             try:
                 if sys.platform.startswith("linux"):
-                    ps = subprocess.Popen(['ps', '-C', 'mysqld', '-o', 'pid'],
-                                          stdout=subprocess.PIPE, close_fds=True).communicate()[0]
-                    pslines = ps.strip().split('\n')
+                    ps, err, rtcode = get_subprocess_output(['ps', '-C', 'mysqld', '-o', 'pid'], self.log)
+                    pslines = ps.strip().splitlines()
                     # First line is header, second line is mysql pid
                     if len(pslines) == 2 and pslines[1] != '':
```

Comment:
Nitpick: Now that we're using `splitlines()` instead of `split("\n")` we don't need to test `pslines[1] != ''` anymore, right? (as `splitlines()` removes the last line when it's empty) :)